src/nwbinspector/checks/nwbfile_metadata.py

```diff
@@ -197,10 +197,12 @@ def check_subject_id_exists(subject: Subject):
 
 @register_check(importance=Importance.BEST_PRACTICE_SUGGESTION, neurodata_type=Subject)
 def check_subject_sex(subject: Subject):
-    """Check if the subject sex has been specified, if one exists."""
+    """Check that the subject sex has been correctly specified."""
     if subject and not subject.sex:
         return InspectorMessage(message="Subject.sex is missing.")
-    elif subject.sex not in ("M", "F", "O", "U"):
+    if subject.species in ("Caenorhabditis elegans", "C. elegans") and subject.sex not in ("XO", "XX"):
+        return InspectorMessage(message="For C. elegans, Subject.sex should be 'XO' or 'XX'.")
+    if subject.sex not in ("M", "F", "O", "U"):
```

Comment:
```suggestion
    elif subject.sex not in ("M", "F", "O", "U"):
```

Reply:
why change to elif? Won't this only come into play when the above condition is false?

Reply:
I think if subject.sex is "XX", the second if will trigger.

Reply:
Ah, you're right - the elif isn't sufficient either.